plugins/processors/starlark/testdata/dropfields_by_type.star

```diff
@@ -0,0 +1,19 @@
+# Drop fields according to their type.
+#
+# To demonstrate this with available types, this script uses the tag "type" to determine which field-types should be
+# dropped. If no such tag exists we assume "string".
+#
+# Example Input:
+# measurement,host=hostname a=1,b="somestring" 1597255410000000000
+#
+# Example Output:
+# measurement,host=hostname a=1 1597255410000000000
+
+def apply(metric):
+    drop = metric.tags.get("type", "string")
```

Comment:
This seems like an unusual example. I don't think I'd expect metrics to contain instructions about what field type to drop. It might be a clearer example if it was just assumed that it's dropping string fields.